code/game/objects/items/devices/chameleonproj.dm

```diff
@@ -190,7 +190,7 @@
 /obj/item/chameleon/prophunt/disrupt(delete_dummy)
 	var/caught = FALSE
 	if(active_dummy)
-		if(gamemaster.gamestate == PROPHUNT_GAME)
+		if(gamemaster.game_state == 4)
```

Comment:
i really could not figure out the issue with this, by the way. we don't undefine PROPHUNT_GAME so I really couldn't figure out why it was acting like this